privaterelay/settings.py

```diff
@@ -12,7 +12,7 @@
 
 from __future__ import annotations
 from pathlib import Path
-from typing import Any, Optional, TYPE_CHECKING
+from typing import Any, Literal, Optional, TYPE_CHECKING, cast, get_args
```

Comment:
suggestion (non-blocking): change L719 to `sentry_release: str | None` and remove import of `Optional`